elfi/examples/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from elfi.examples import ma2
+from elfi.examples import bdm
```

Comment:
I guess people can pick with specific imports if they want. There are some places in the code and examples where both are used.